### ISSUE TITLE
训练en_PP-OCRv3_rec 报错，没有提示什么错误

### ISSUE THREAD

**Issue by cqray1990** [PaddlePaddle/PaddleOCR#7646]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment： ubantu 18
- 版本号/Version：Paddle： 2.3.2 cuda 10.2,cudnn 8.2
-  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：python3 -m paddle.distributed.launch --log_dir=./debug/ --gpus '0,1'  tools/train.py -c configs/rec/PP-OCRv3/en_PP-OCRv3_rec.yml -o Global.pretrained_model=./pretrain_models/en_PP-OCRv3_rec_train/best_accuracy

但是训练：
python3 -m paddle.distributed.launch --log_dir=./debug/ --gpus '0,1'  tools/train.py -c configs/rec/rec_icdar15_train.yml
没有问题
- 完整报错/Complete Error Message：

2022/09/19 17:43:45] ppocr INFO:         shuffle : True
[2022/09/19 17:43:45] ppocr INFO: profiler_options : None
[2022/09/19 17:43:45] ppocr INFO: train with paddle 2.3.2 and device Place(gpu:0)
I0919 17:43:45.872455  9964 nccl_context.cc:83] init nccl context nranks: 2 local rank: 0 gpu id: 0 ring id: 0
W0919 17:43:46.172901  9964 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.2, Runtime API Version: 10.2
W0919 17:43:46.174533  9964 gpu_resources.cc:91] device: 0, cuDNN Version: 8.2.
[2022/09/19 17:43:46] ppocr INFO: Initialize indexs of datasets:train_data/ic15_data/rec_gt_train.txt']
[2022/09/19 17:43:46] ppocr INFO: Initialize indexs of datasets:['train_data/ic15_data/rec_gt_test.txt']
[2022/09/19 17:43:46] ppocr INFO: train dataloader has 17 iters
[2022/09/19 17:43:46] ppocr INFO: valid dataloader has 17 iters
[2022/09/19 17:43:46] ppocr INFO: train from scratch
[2022/09/19 17:43:46] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations
INFO 2022-09-19 17:44:02,156 launch_utils.py:322] terminate process group gid:9964
INFO 2022-09-19 17:44:02,156 launch_utils.py:322] terminate process group gid:9964
INFO 2022-09-19 17:44:09,160 launch_utils.py:343] terminate all the procs
INFO 2022-09-19 17:44:09,160 launch_utils.py:343] terminate all the procs
ERROR 2022-09-19 17:44:09,161 launch_utils.py:642] ABORT!!! Out of all 2 trainers, the trainer process with rank=[1] was aborted. Please check its log.
ERROR 2022-09-19 17:44:09,161 launch_utils.py:642] ABORT!!! Out of all 2 trainers, the trainer process with rank=[1] was aborted. Please check its log.
INFO 2022-09-19 17:44:13,163 launch_utils.py:343] terminate all the procs
INFO 2022-09-19 17:44:13,163 launch_utils.py:343] terminate all the procs
INFO 2022-09-19 17:44:13,163 launch.py:402] Local processes completed.
INFO 2022-09-19 17:44:13,163 launch.py:402] Local processes completed.
